### ISSUE TITLE
get github pages working

### ISSUE THREAD

**Issue by willjrowe** [willjrowe/IR-Plan-Page#1]
so i cloned your fork and ran npm install and then npm run deploy and now I'm getting the same hanging issue. What are good next steps? I don't want to get too messed up with my branches again lol

**Comment by sa-webb** [comment]
I agree. Here is my next guess - for some reason it isn't being allowed to publish it on GitHub. Therefore, I'm going to look through my account settings to see if there's some permission that I may have that you don't.

**Comment by sa-webb** [comment]
Do you have an SSH key configured with your account? My guess is that maybe GitHub pages isn't making it to publish stage because it isn't authorized but it doesn't have the options to "login" or something. 

My exact steps to get that to work were 
1. I forked your repo - from master branch
2. I checked that you had gh-pages as dev dependency
3. I checked that you had pathPrefix
4. I CHANGED the deploy script "deploy": "gatsby build --prefix-paths && gh-pages -d public"
5. I executed that command.
Which published for me - then I got into the issue of it displaying the README but we can resolve that easily.
So for some reason that command stalls and doesn't let you publish which leads me to believe somehow where you are publishing from is unauthenticated.

**Comment by willjrowe** [comment]
Ok sounds good.

I am able to `git push` and `git pull` without having to input my credentials 

and `git remote -v` returns the correct repo address

**Comment by willjrowe** [comment]
Sorry I keep commenting right after you do, looking at it now.

**Comment by sa-webb** [comment]
All good

**Comment by willjrowe** [comment]
I don't have any SSH keys. :(

However I might have figured something out. The first time I ran npm run deploy the process hung. However, the second and third time I ran npm run deploy I got the error about the gh-pages branch already existing (however I cannot find any evidence of the branch existing either locally or remotely). So my thoughts are perhaps the process was actually initially hanging and I just terminated it too early. Not sure how to reset this though.

Also might try this: https://www.npmjs.com/package/gh-pages#debugging
Not sure how difficult that would be to set up though

**Comment by sa-webb** [comment]
Keep trying! Also, if you just need it to be published then I recommend Netlify. They have a free option and it's easy. Keep me posted though, I will leave my fork and publish available until otherwise. Goodluck!
